build/archives/BUILD.gn

```diff
@@ -280,3 +280,16 @@ if (is_mac) {
     ]
   }
 }
+
+if (is_win) {
+  zip_bundle("archive_win_gen_snapshot") {
+    deps = [ "//third_party/dart/runtime/bin:gen_snapshot" ]
+    output = "$target_platform_name-$android_cpu-$flutter_runtime_mode/windows-x64.zip"
```

Comment:
@pbo-linaro is working on adding Arm64 support to Windows (see https://github.com/flutter/flutter/issues/62597 and https://github.com/flutter/engine/pull/35962). Do we need to anything for arm64 artifacts here?

Reply:
dart-sdk for windows-arm64 is building fine (previous work before flutter), so that could be nice to prepare this part in this PR if possible.

Reply:
This is also covering windows arm64.